### PR TITLE
LED-186 User Manual

### PR DESCRIPTION
**Pull request by richarel**
I'm opening a PR for this now for you all to review for grammatical errors, things I may have forgotten, etc. I'm aware that some of the screenshots will need updated after everyone gets their stuff in, and it will probably need updated pretty frequently for the rest of the quarter.

No rush on this. Don't bore yourself to tears reading it.

### CODE REVIEW

**Review by shellajt — CHANGES_REQUESTED** [review]
First off, this looks incredible. Good work! I've got a couple suggestions, but feel free to take care of them next week or after break. This looks great for a week's work.

It could use a section detailing how we store the last used database file. That could easily be confusing for new users.

I think we should mention that data is encrypted when we discuss logging in

Font size seems to change in the middle of some of the paragraphs or isn't consistent between the paragraphs.

Undo is mentioned by not Redo.

**Review by cjm721 — CHANGES_REQUESTED** [review]
* On `Creating a New Database File and Password` The window shown will change on OS. Currently have similar window, but Linux / Mac one can look a lot different (Don't think this is really an issue at the same time)
  * Also should mention no way to recover lost passwords.
* When explaining what is in the Tabs mention that the way the buttons work will be explained latter (./ have a link to them)
* Mention Export is also encrypted with the same password
* Import Transactions. Might want to reword this as we are able 
* Update Ignored Transaction Page Image
  * Also instead of exiting out of it, hit the done button (new item)
* When adding transaction via TableView, if column is sorted that they are editing once they hit enter it will move to the order position
* `Automatically Adding Tags to Imported Transactions` Automatically adds Tags to transactions  (not payees)
* `Filtering Transactions` Might want to call this organizing transactions. The change of the column sort does not filter anything. 
* Undo will also pull up a confirmation popup that will tell you what you are going to undo and if you really want to do that.
* Mention `Exporting Data` twice. The first time is just a little more vague. Might want to just link the description to the second one.

**Review by shellajt — APPROVED** [review]

**Review by cjm721 — APPROVED** [review]
